jupyter-on-gke/variables.tf

```diff
@@ -70,4 +70,10 @@ variable "url_domain_name" {
   type = string
   description = "Name of the domain provided by the user. This var will only be used if url_domain_addr is not empty"
   default = ""
+}
+
+variable "support_email" {
+  type = string
+  description = "Email for users to contact with questions about their consent"
```

Comment:
Still think that the admin setting up this environment needs to know where to find this email. Can we provide either a link to the Pantheon oauth consent page where they can find it, or instructions for how to find it?

Reply:
Ohh, I see - we just associate whatever email they provide with the brand we create? In which case, ignore this :)